gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -423,7 +423,10 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
       model_num = gst_tensor_filter_parse_modelpaths_string (prop, model_files);
       if (model_num == 1) {
         if (!g_file_test (prop->model_file, G_FILE_TEST_IS_REGULAR))
-          g_critical ("Cannot find the model file: %s\n", prop->model_file);
+          g_critical ("Cannot find the model file: %s\n"
+              "Ignore this if you have specified name and the model is preloaded.\n"
+              "Note that this should be refactored.\n", prop->model_file);
+          /** @todo This mechanism should be refactored. */
```

Comment:
is this the case for the frameworks which have set `run_without_model` as TRUE?

Reply:
Nope.

It's just not a path, but a name.